apps/ejabberd/test/acc_SUITE.erl

```diff
@@ -38,3 +38,36 @@ store_and_retrieve(_C) ->
     ok.
 
 
+init_from_element(_C) ->
+    Elem = {xmlel, <<"iq">>,
```

Comment:
Thanks @gadgetci for this! @bartekgorny could you remove the duplciated code? Looks like the stanzas are exactly the same.